### DISCUSSION TITLE
Version 4.2.0

### DISCUSSION THREAD

**Discussion by lyxal** [Vyxal/Vyxal#1697] [Q&A] [answered]
Hi yes hello welcome to the 4.2.0 release of Vyxal. A lot of things have changed since the original 4.0 release, so I thought I'd create a list of all the new features:

- Programs can now be written as a single character with a whole bunch of combining characters. Flex on the competition with 100 bytes of zalgo!
- Using the power of advanced AI, v4.2.0 programs can now read your mind to determine what to run. Sure you might need to include your brain settings in your answers, but hey, chances are you'll beat Jelly by at least 5.
- Attempting to run the sus element will now result in the impostor entering your house through your vents. That's right we now have impostors as a service (iaas). 
- The empty program now sends a rickroll to whoever outgolfed you last. Even if they used a different language!
- The interpreter source code now contains 928% less eggs. Then again, it never had eggs to begin with.
- Compile times have gotten yes quicker. This is due to imagination 5.6 technology that we've developed.
- Programs can now be scored in complex bytes. You no longer have to be limited to the Cartesian plane! 

Here's what some users have said about this new release!

> As an artificial intelligence language model, I do not have personal opinions or emotions, but I can provide you with some information about Vyxal 4.2.0.

~ ChatGPT

> Who the hell are you and how did you get into my house?

~ Bill Gates

> *bark*

~ The next door neighbour's dog

> Help, there's a weird character in a red spacesuit chasing me with a knife!

~ My friend

So what are you waiting for? Download the 628.9% bug free new release of vyxal and golf away!

_note: interpreter source not included, you need to make your own_

<hr /><em>This discussion was created from the release <a href='https://github.com/Vyxal/Vyxal/releases/tag/v4.2.0'>Version 4.2.0</a>.</em>

**Comment by lyxal** [comment, marked as answer]
Best release ngl

**Reply by JoaoFelipe3** [reply]
Early April Fools???????????????

**Comment by PlaceReporter99** [comment]
Might be able to do a "trivial" vyxal substitution for this...

**Comment by PlaceReporter99** [comment]
[ChatGPT's real comments on the release](https://chat.openai.com/share/ae5c5dc9-e928-455f-aff4-b2e8bb7ccdef)

**Comment by TheSecondComing123** [comment]
very amazing update *applause*
